configs/kubernetes-nightly-configmap.yaml

```diff
@@ -4,9 +4,10 @@ metadata:
   name: publisher-config
 data:
   config: |
-    source-org: kubernetes
+    source-org: liggitt
     source-repo: kubernetes
+    base-package: k8s.io
     target-org: kubernetes-nightly
-    rules-file: https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/publishing/rules.yaml
+    rules-file: https://raw.githubusercontent.com/sttts/kubernetes/sttts-go-mod-rules/staging/publishing/rules.yaml
```

Comment:
do I need to fold this into my PR?

Reply:
added smoke-test changes in https://github.com/kubernetes/kubernetes/pull/74877/commits/b3527f41f01f900954e378eb8d78a0b5a72dbfde